### PR TITLE
feat: add execution tags by env vars

### PR DESCRIPTION
**Pull request by saartochner**

### CODE REVIEW

**Review by OrrLumigo — APPROVED** [review]
COOL FEATURE!!

File: src/tracer.test.js [hunk 1]
```diff
@@ -463,6 +464,22 @@ describe('tracer', () => {
...
+    expect(spies.setSwitchOff).not.toHaveBeenCalled();
+
+    const { context } = new HandlerInputesBuilder().build();
+
+    const userHandler4 = async (event, context, callback) => {
+      return {};
+    };
+    await lumigoTracer.trace(userHandler4)({ key1: 'aaa' }, context, jest.fn());
+
+    expect(spies.autoTagEvent).toBeCalledWith({ key1: 'aaa' });
```
Comment (doriaviram) [inline src/tracer.test.js:480]: Don't use the mock, query the ExecutionTags API and check if there is a tag

File: src/globals.ts [hunk 2]
```diff
@@ -111,7 +111,11 @@ export const ExecutionTags = (() => {
   // @ts-ignore
   const clear = () => (global.tags = []);
 
-  return { addTag, getTags, clear, validateTag };
+  const autoTagEvent = (event) => {
+    getAutoTagKeys().forEach((key) => event[key] && addTag(key, event[key]));
```
Comment (doriaviram) [inline src/globals.ts:115]: Make sure that the `value` is string

File: src/tracer.test.js (hunk 1 above)
Comment (saartochner) [reply src/tracer.test.js:480]: Unfortunately this is hard :(

**Review by doriaviram — APPROVED** [review]

**Pull request merged**